test.sh

```diff
@@ -2,38 +2,21 @@
 
 set -euxfo pipefail;
 
+source "_common.sh";
+
 test-image () {
-  declare -r version="${1}";
-  declare -r image="jackfirth/racket:${version}";
-  docker container run -it "${image}" racket -e "(+ 1 2 3)";
-  docker container run -it "${image}" raco pkg install --auto rackunit-lib;
-};
+  declare -r image="${1}";
 
-foreach () {
-  declare -r command="${1}";
-  declare -r args="${@:2}";
-  for _arg in ${args}; do
-    "${command}" "${_arg}";
-  done;
+  if echo "${image}" | grep "\-full" -q; then
+      test_package="deta-lib";  # rackunit-lib is already installed in the -full variant
```

Comment:
Instead of conditionally installing `deta-lib` or `rackunit-lib`, lets just always install `deta-lib`. That way we don't have to worry about installing something that could already be included in the image.

I do wish there was some sort of canonical `example` package in the package catalog. That way this test script wouldn't be dependent on whether `deta-lib`, an (I assume) actively maintained package, has any build errors.

Reply:
Good point! I think I'll go ahead and create a package like that and then use it here. I hadn't considered what would happen were I to break the build for `deta`.